http-api/src/v1/sessions.rs

```diff
@@ -0,0 +1,177 @@
+use crate::auth::{AuthRequest, AuthState, Session};
+use crate::Context;
+use crate::Error;
+
+use std::collections::HashMap;
+use std::convert::TryInto;
+use std::env;
+use std::iter::repeat_with;
+use std::str::FromStr;
+use std::time::Duration;
+
+use axum::{
+    extract::Path,
+    response::IntoResponse,
+    routing::{get, post, put},
+    Extension, Json, Router,
+};
+use chrono::{DateTime, Utc};
+use ethers_core::utils::hex;
+use hyper::http::uri::Authority;
+use serde_json::json;
+use siwe::Message;
+
+pub const UNAUTHORIZED_SESSIONS_EXPIRATION: Duration = Duration::from_secs(60);
+
+/*
+ *
+fn session_filters(ctx: Context) -> BoxedFilter<(impl Reply,)> {
+    session_create_filter(ctx.clone())
+        .or(session_signin_filter(ctx.clone()))
+        .or(session_get_filter(ctx))
+        .boxed()
+}
+
+/// `POST /sessions`
+fn session_create_filter(
+    ctx: Context,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    warp::post()
+        .map(move || ctx.clone())
+        .and(path::end())
+        .and_then(session_create_handler)
+}
+
+/// `PUT /sessions/:session-id`
+fn session_signin_filter(
+    ctx: Context,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    warp::put()
+        .map(move || ctx.clone())
+        .and(path::param::<String>())
+        .and(path::end())
+        .and(warp::body::json())
+        .and_then(session_signin_handler)
+}
+
+/// `GET /sessions/:session-id`
+fn session_get_filter(
+    ctx: Context,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    warp::get()
+        .map(move || ctx.clone())
+        .and(path::param::<String>())
+        .and(path::end())
+        .and_then(session_get_handler)
+}
+*/
+
+pub fn router(ctx: Context) -> Router {
+    Router::new()
+        .route("/sessions", post(session_create_handler))
+        .route(
+            "/sessions/:session_id",
+            get(session_get_handler).put(session_signin_handler),
+        )
+        .layer(Extension(ctx.clone()))
+}
+
+/// TODO: Add description.
```

Comment:
Some handlers are missing description. We want the same format above each handler, right?

Reply:
Also what should the description be for:
- `GET /sessions/:session_id`
- `PUT /sessions/:session_id`
- `POST /sessions`

Reply:
Get session, update session, create session.